app/view_objects/daily_progress_digest.rb

```diff
@@ -0,0 +1,36 @@
+class DailyProgressDigest
+
+  def initialize params
+    @params = params
+  end
```

Comment:
@vlasiak, I suggest to use `options` instead of `params` (it is implementation pattern, like convention).